packages/web/src/pages/upload-page/components/SubmitRowAnchored.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  HarmonyButton,
+  HarmonyButtonSize,
+  HarmonyButtonType,
+  IconUpload
+} from '@audius/stems'
+
+import styles from './SubmitRowAnchored.module.css'
+
+const messages = {
+  complete: 'Complete Upload'
+}
+
+export const SubmitRowAnchored = () => {
+  return (
+    <>
+      <div className={styles.buttonRow}>
+        <HarmonyButton
+          text={messages.complete}
+          variant={HarmonyButtonType.PRIMARY}
+          size={HarmonyButtonSize.DEFAULT}
+          iconRight={IconUpload}
+        />
+      </div>
+      <div className={styles.placeholder} />
```

Comment:
How come we need the placeholder if the submit row is anchored with bottom specifying play bar height?

Reply:
It's a spacer that keeps the scrolling content from being hidden by the overlay

Reply:
you could also use position sticky with bottom 0 to have it take up the space that it would without needing this